docs/4.3. Behaviour - Tokens.md

```diff
@@ -97,28 +107,23 @@ access to any of the APIs. A missing API key MUST indicate no access to that spe
 }
 ```
 
-In addition, a set of universal `key:value` pairs MAY be included for each API, with the `keys` indicating the type
-of access to the API and the `value` indicating individual paths that are permitted to be accessed in such a way.
-Wild-carding is permitted in the path specifiers. Individual AMWA specifications MAY also specify additional
-key:value pairs specific to the API. General access keys include:
-*   **create** - ability to insert a new resource. This usually takes the form of POST HTTP request verbs.
+The `value` for each NMOS API indicates the access rights of the given user for that specific API. The value required
+by the Resource Server MUST exactly match one of the values in the array. General access rights include:
+*   **write** - ability to insert a new, or edit an existing, resource. This usually takes the form of POST, PUT or
+    PATCH HTTP request verbs.
 *   **read** - ability to retrieve the resource. This usually takes the form of GET and HEAD HTTP request verbs.
-*   **update** - ability to change an existing resource. This usually takes the form of PUT or PATCH HTTP request verbs.
-*   **delete** - ability to remove an existing resource. This usually takes the form of the DELETE HTTP request verbs.
 
-An example of a `x-nmos-api` claim is shown below.
+Individual AMWA specifications MAY specify additional values specific to the individual API.
+
+An example of a `x-nmos-api` claim is shown below. This would permit querying an IS-04 Registry, and making
+connections via an IS-05 connection management interface, but would not permit the user to register a resource with
+an IS-04 Registry.
 
 ```json
 "x-nmos-api": {
-  "register": {
-    "read": ["node.*", "query.*"],
-    "create": ["query.subscriptions", "registration.*"]
-  },
-  "connection": {
-    "update": {
-      "resources": ["senders.2d2ed05a-4b2e-4cc9-8e8c-cc62a27db97a"]
-    }
-  }
+  "register": ["read"],
+  "query": ["read", "write"],
```

Comment:
I was thinking that write should imply read, but perhaps it shouldn't - I'm not sure. We should have some text that indicates whether it does or not either way, and if it does imply read then indicate that inclusion of 'read' in addition is optional.

I can't easily think of a case where you would allow write-only (given that this includes delete), and given that many write requests require use of options (which would otherwise be read) it may make no sense to give write without read.

Reply:
I think, with us being Python developers, Explicit is better than implicit? 
Also Linux file permissions separate the two?
We can always change it back